disk/tests/test_check.py

```diff
@@ -17,3 +22,18 @@ def test_check(aggregator, instance_basic_volume, gauge_metrics, rate_metrics):
         aggregator.assert_metric(name)
 
     aggregator.assert_all_metrics_covered()
+
+
+@pytest.mark.integration
+@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')
+@pytest.mark.usefixtures('psutil_mocks')
+def test_instance_with_blkid_cache_file(aggregator, instance_blkid_cache_file):
```

Comment:
Since we are mocking things. I think it's preferable to move this test to `test_unit.py`.

Once move to `test_unit.py`, those two decorators are probably not needed.
```
@pytest.mark.integration
@pytest.mark.skipif(sys.platform != 'linux', reason='disk labels are only available on Linux')
```

Reply:
I just had to leave the skipif since there is a condition in the code that gates the label feature only if the platform is set to linux: https://github.com/DataDog/integrations-core/blob/6530efe61b25ca66df1c6c49ad7259fcb682ab53/disk/datadog_checks/disk/disk.py#L59